Cargo.toml

```diff
@@ -3,11 +3,13 @@ resolver = "2"
 members = [
     "docstrings",
     "forc",
+    "forc-client-ops",
```

Comment:
Instead of adding an extra `forc-client-ops` crate, perhaps we can add a lib.rs and `[[lib]]` target to the `forc-client` crate itself? Then we can add an `ops` submodule if necessary.